cluster-sim/cluster_sim/locale/ru/LC_MESSAGE/cluster_sim.po

```diff
@@ -0,0 +1,966 @@
+msgid "Deregistered agent: %s"
+msgstr "Отменена регистрация агента: %s"
+
+#: fake\_hsm\_coordinator.py:303
+msgid "Got start on %s from %s"
+msgstr "Запустился на %s из %s"
+
+#: fake\_hsm\_coordinator.py:307
+msgid "Got finish on %s from %s"
+msgstr "Закончил на %s из %s"
+
+#: fake\_hsm\_coordinator.py:348
+msgid "Unknown HSM Coordinator control value: %s"
+msgstr "Неизвестное управляющее значение Координатора HSM: %s"
+
+#: fake\_hsm\_coordinator.py:355
+msgid "Starting HSM Coordinator for %s"
+msgstr "Запускаю Координатора HSM для %s"
+
+#: fake\_hsm\_coordinator.py:361
+msgid "Stopping HSM Coordinator for %s"
+msgstr "Останавливаю Координатора HSM для %s"
+
+#: fake\_hsm\_copytool.py:38
+msgid "Opened %s for write"
+msgstr "Открыл %s на запись"
+
+#: fake\_hsm\_copytool.py:147
+msgid "Copytool %s started"
+msgstr "Запустился копировальщик %s"
+
+#: fake\_hsm\_copytool.py:189
+msgid "Attempt to start unregistered copytool: %s"
+msgstr "Попытка апустить незарегистрированный копировальщик: %s"
+
+#: fake\_hsm\_copytool.py:251
+msgid "Starting HSM Copytool: %s"
+msgstr "Запускаю копировальщик HSM: %s"
+
+#: fake\_hsm\_copytool.py:267
+msgid "Timed out waiting for copytool thread to start"
+msgstr "Истекло время ожидания запуска потока копировальщика"
+
+#: fake\_hsm\_copytool.py:271
+msgid "Stopping HSM Copytool: %s"
+msgstr "Останавливаю Копировальщик HSM: %s"
+
+#: fake\_power\_control.py:40
+msgid "Handling PDU request from %s:%s"
+msgstr "Обрабатываю запрос РЩП от %s:%s"
+
+#: fake\_power\_control.py:57
+msgid "Creating PDU server for %s on %s:%s"
+msgstr "Создаю сервер РЩП для %s на %s:%s"
+
+#: fake\_power\_control.py:102 fake\_power\_control.py:158
+msgid "%s: received connection from %s:%s"
+msgstr "%s: получил соединение от %s:%s"
+
+#: fake\_power\_control.py:134
+msgid "POWER: Toggled %s:%s to %s"
+msgstr "ПИТАНИЕ: Переключил %s:%s в %s"
+
+#: fake\_power\_control.py:167
+msgid "%s: client %s:%s disconnected"
+msgstr "%s: клиент %s:%s отвалился"
+
+#: fake\_power\_control.py:219
+msgid "Unhandled response in %s: '%s' (%s)"
+msgstr "Необработанный ответ в %s: '%s' (%s)"
+
+#: fake\_power\_control.py:246
+msgid "Received username: %s"
+msgstr "Получил имя: %s"
+
+#: fake\_power\_control.py:251
+msgid "Received password: %s"
+msgstr "Получил пароль: %s"
+
+#: fake\_power\_control.py:255
+msgid ""
+"\n"
+"\n"
+"\n"
+"American Power Conversion               Network Management Card AOS      v3.7.3\n"
+"(c) Copyright 2009 All Rights Reserved  Rack PDU APP                     v3.7.3\n"
+"-------------------------------------------------------------------------------\n"
+"Name      : RackPDU                                   Date : 02/13/2013\n"
+"Contact   : Unknown                                   Time : 10:49:37\n"
+"Location  : Unknown                                   User : Administrator\n"
+"Up Time   : 7 Days 15 Hours 31 Minutes                Stat : P+ N+ A+\n"
+"\n"
+"Switched Rack PDU: Communication Established\n"
+msgstr ""
```

Comment:
Why is the `msgstr` empty here?  Will this result in an empty string being assigned to `banner` at [fake_power_control.py:255](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/354/files#diff-d1460e059877e95da2e0c3bee77682caR255)?

Reply:
Perhaps, it's a typo or just the current end-of-translation (I've translated about 1/3 so far).